environments/backfillStorage.toml

```diff
@@ -151,10 +151,14 @@
         deployed = 8928198
-    [contract.MCD_CAT]
+    [contract.MCD_CAT_1.0.0]
```

Comment:
I know this isn't _really_ a responsibility of this PR, but it'd be cool to take the transformers out of here + plugins/backfill/storage/transformerExporter.go to avoid kicking off a redundant backfill process when this is merged